content/scout/policy/ci.md

```diff
@@ -0,0 +1,141 @@
+---
+title: Evaluate policy compliance in CI
+description: |
+  Configure your continuous integration pipelines to fail
+  when Policy Evaluation for an image is worse compared to baseline
+keywords: scout, supply chain, policy, ci
+---
+
+Adding Policy Evaluation to your continuous integration pipelines helps you
+detect and prevent cases where code changes would cause policy status become
+worse compared to your baseline.
+
+The recommended strategy for Policy Evaluation in a CI setting involves
+evaluating a local image and comparing the results to a baseline. If the policy
+compliance for the local image is worse than the specified baseline, the
+CI run fails with an error. If the policy compliance is unchanged or improved,
+the CI run succeeds.
+
+This comparison is relative, meaning that it's only concerned with whether your
+CI image is better or worse than your baseline. It's not an absolute check to
+pass or fail all policies. By measuring relative to a baseline that you define,
+you can quickly see if a change has a positive or negative impact on policy
+compliance.
+
+## How it works
+
+When you do Policy Evaluation in CI, you run a local policy evaluation on the
+image you build in your CI pipeline. To run a local evaluation, the image that
+you evaluate must exist in the image store where your CI workflow is being run.
+Either build or pull the image, and then run the evaluation.
+
+To run policy evaluation and trigger failure if compliance for your local image
+is worse than your comparison baseline, you need to specify the image version
+to use as a baseline. You can hard-code a specific image reference, but a
+better solution is to use [environments](../integrations/environment/_index.md)
+to automatically infer the image version from an environment. The examples that
+follow use environments to compare the CI image with the image in the
+`production` environment.
+
+## Example
+
+The following example on how to run policy evaluation in CI uses the [Docker
+Scout GitHub Action](https://github.com/marketplace/actions/docker-scout) to
+execute the `compare` command on an image built in CI. The compare command has
+a `to-env` input, which will run the comparison against an environment called
+`production`. The `exit-on` input is set to `policy`, meaning that the
+comparison fails only if policy compliance has worsened.
+
+This example doesn't assume that you're using Docker Hub as your container
+registry. As a result, this workflow uses the `docker/login-action` twice:
+
+- Once for authenticating to your container registry.
+- Once more for authenticating to Docker to pull the analysis results of your
```

Comment:
Did you mean Scout?
```suggestion
- Once more for authenticating to the Docker Scout platform to pull the analysis results of your
```

Reply:
Not really, we are authenticating to Docker, nothing specific to Scout (see the sentence that follows - if you use Hub you only need to auth once)